flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/JobLeaderService.java

```diff
@@ -432,8 +432,8 @@ protected void onRegistrationFailure(Throwable failure) {
 		@Override
 		protected CompletableFuture<RegistrationResponse> invokeRegistration(
 				JobMasterGateway gateway,
-				JobMasterId jobMasterId,
-				long timeoutMillis) throws Exception {
+				JobMasterId fencingToken,
```

Comment:
Most where we will call it `jobMasterId` though. I'm ok with this change.

Reply:
Ok, then I keep as is because it is defined with this name in the super class.